content/blog/posts/New-Project-April2017.rst

```diff
@@ -0,0 +1,14 @@
+New Project: New ftp Site for libpng Tarball Distributions
+==========================================================
+:date: 2017-04-28
+:slug: new-project-ftp
+:author: Amanda Kelner
+:img: OSL-Project-Announcement-Picture.png
+:tag: featured-stories
+
+The OSL took on a new project and delivers a new ftp site for libpng tarball
```

Comment:
"... delivers a new mirroring site libpng source ..."

Also, I would add a link to the libpng homepage: http://www.libpng.org/pub/png/libpng.html